src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java

```diff
@@ -124,8 +129,14 @@ public boolean isPaperResponse() {
             return false;
         }
 
+        //return Objects.nonNull(caseData.getTakenOfflineDate()) && Objects.nonNull(caseData.getCcdState())
```

Comment:
do we need to have it here if its been commented out?

Reply:
removed